test/storybook.test.js

```diff
@@ -176,6 +176,16 @@ describe('percy storybook', () => {
     ]);
   });
 
+  it('sends the version of storybook when creating snapshots', async () => {
+    await Storybook.run(['http://localhost:9000']);
+
+    expect(logger.stderr).toEqual([]);
+    expect(mockAPI.requests['/builds/123/snapshots'].map(req => req.headers['user-agent'])[0])
+      .toMatch(
+        /^Percy\/v1 @percy\/client\/\S+ @percy\/storybook\/\d+\.\d+\.\d+.*? \(storybook\/\d+\.\d+\.\d+.*?; node\/v[\d.]+.*\)$/
+      );
```

Comment:
I don't think we need to necessarily test the entire user-agent. Just that it contains what we intend it to.

```suggestion
      .toMatch(/storybook\/\d+\.\d+\.\d+.*?;/);
```